app/index.js

```diff
@@ -65,11 +67,12 @@ module.exports = yeoman.Base.extend({
       this.option('version', {
         desc: g.f('Display version information'),
```

Comment:
I noticed the first argument, `name`, is using `description` property name; while all other options (not arguments!) are still using `desc` property name. Could you please check which form is the right one and ideally unify the implementation to use the same property name everywhere (if possible)?

Reply:
@bajtos either of them works, see [yeoman-generator](https://github.com/yeoman/generator/blob/1cd1031eb1cc2ed0288265dd15ddc6cfbcffabb7/lib/index.js#L202)

And our [help message generator](https://github.com/strongloop/generator-loopback/pull/363/files#diff-2dc44d3980a09a85c3906ab18870bca8R109) 

But I agree that it's better to unify them. I will change `description` to `desc`.